### PR TITLE
Eslint fixes

### PR DESCRIPTION
**Pull request by joshbenaron**
Hey,

Currently, the project doesn't use eslint correctly which make it near impossible to use in a lot of IDEs (as you can see below). Also it provides a lot of helpers for reducing logical errors. I'm going to be making project wide changes to fix formatting.

image

### CODE REVIEW

**Comment by joshbenaron** [comment]
I have around 60 more errors to fix

**Comment by joshbenaron** [comment]
There's one error remaining which actually has functional implications @hlolli 
You'll see there is the same case label on two cases - which was picked up by eslint! Can you tell me which one is correct so I can make the last error fixing change and this can be merged. 
This is from [this file and line](https://github.com/ArweaveTeam/vartex/blob/main/src/database/cassandra.database.ts#L196)
```ts
    case 'block_size':
    case 'diff':
    case 'height':
    case 'last_retarget': // HERE
    case 'reward_pool':
    case 'timestamp':
    case 'weave_size': {
      return toLong(obj[key]);
    }
    case 'cumulative_diff':
    case 'hash':
    case 'hash_list_merkle':
    case 'indep_hash':
    case 'nonce':
    case 'previous_block':
    case 'reward_addr':
    case 'tx_root':
    case 'wallet_list':
    case 'last_retarget':  // HERE
    {
      if (obj[key] || isNumeric(obj[key])) {
        return typeof obj[key] === 'string' ? obj[key] : obj[key].toString();
      } else {
        return '';
      }
    }```

**Comment by hlolli** [comment]
Very needed! Will review in a sec, fixing the test I ruined this morning... you could pull and add a lint chek test run actually after that is done, to have green stamp on the test as well as lint. I could also actually push to your branch if you don't mind?

**Comment by joshbenaron** [comment]
I can add prettier but as far as I'm aware there are some rules excluded in prettier e.g. curly bracket rules

**Comment by hlolli** [comment]
> @hlolli I can add prettier but as far as I'm aware there are some rules excluded in prettier e.g. curly bracket rules

correct, so in that regard, eslint is fine anything that's not a prettier rule, imo.

**Comment by joshbenaron** [comment]
> > @hlolli I can add prettier but as far as I'm aware there are some rules excluded in prettier e.g. curly bracket rules
> 
> correct, so in that regard, eslint is fine anything that's not a prettier rule, imo.

Nice - I'll add prettier too! Can you let me know what's the correct case for my comment [here](https://github.com/ArweaveTeam/vartex/pull/1#issuecomment-894643550)

**Comment by joshbenaron** [comment]
I see you've now made another commit to the main branch. This has created a lot of conflicts with the formatting again... Could we resolve this first?

**Comment by hlolli** [comment]
> @hlolli I see you've now made another commit to the main branch. This has created a lot of conflicts with the formatting again... Could we resolve this first?

Yes I can take a look asap. I had to leave early last night so I wanted to push. And I see now that gql on current legacy-prod is off, so I'll look into that first then fix the conflicts 👍

**Comment by hlolli** [comment]
done, if this happens again you can do this (if you're on main on your fork with uncommited changes)

```
git remote add upstream https://github.com/ArweaveTeam/vartex.git
git fetch upstream
git merge upstream/main --strategy-option theirs
git push origin main
```

**Comment by joshbenaron** [comment]
@hlolli please can you review this case statement. [here](https://github.com/ArweaveTeam/vartex/blob/main/src/database/cassandra.database.ts#L204)
There's a duplicate label `last_retarget` on both

**Comment by hlolli** [comment]
> @hlolli please can you review this case statement. [here](https://github.com/ArweaveTeam/vartex/blob/main/src/database/cassandra.database.ts#L204)
> There's a duplicate label `last_retarget` on both

good catch, it should only be the first one as it's stored as `last_retarget bigint` or in js-land, as number toLong. So the second case of last_retarget can be deleted.

File: src/database/sync.database.ts [hunk 1]
```diff
@@ -1,96 +1,80 @@
...
-    ? undefined
-    : parseInt(process.env['DEVELOPMENT_SYNC_LENGTH'] as string);
+  !process.env["DEVELOPMENT_SYNC_LENGTH"] ||
+  R.isEmpty(process.env["DEVELOPMENT_SYNC_LENGTH"]) ?
+    undefined :
+    parseInt(process.env["DEVELOPMENT_SYNC_LENGTH"] as string);
 
-if (developmentSyncLength === NaN) {
-  console.error('Development sync range variable produced, illegal value NaN');
+if (isNaN(developmentSyncLength)) {
```
Comment (hlolli) [inline src/database/sync.database.ts:55]: this caused the test to break, the test for isNaN is true on any non-number, whereas === NaN is a check for the primitive value of NaN which is never true for strings as example.

Comment (joshbenaron) [reply src/database/sync.database.ts:55]: Noted - I'm happy with the change

**Pull request merged**